tests/test_dsl.py

```diff
@@ -67,7 +68,7 @@ class OldCohort(OldCohortWithPopulation):
     cohort = cohort_with_population
     events = tables.clinical_events
     cohort.code = (
-        events.filter(codelist(["Code1"], "ctv3").contains(events.code))
+        events.filter(events.code.is_in(codelist(["Code1"], "ctv3")))
```

Comment:
:+1: I prefer this

Reply:
Me too.

Also, we can return to using `make_codelist` in these tests.